### ISSUE TITLE
bug with `string-no-newline`

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#905]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

There are a lot of errors `Unexpected newline in string (string-no-newline)`.

> Which rule, if any, is this issue related to?

`string-no-newline`

> What CSS is needed to reproduce this issue?

e.g.

```scss
p[href^="https://"]:before {
    content: "secure";
}

.foo {
    color: #fff;
}

.bar {
    color: #000;
}
```

In CSS errors also throw.

If we change code on (with one `/`), not errors:
 
```scss
p[href^="https:/"]:before {
    content: "secure";
}

.foo {
    color: #fff;
}

.bar {
    color: #000;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "string-no-newline": true
  }
}
```

> Which version of stylelint are you using?

`5.0.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

PostCSS plugin

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

SCSS

> What did you expect to happen?

Nothing

> What actually happened (e.g. what warnings or errors you are getting)?

"The following warnings were flagged:"

```shell
test.css
1:29    Unexpected newline in string (string-no-newline)
2:23    Unexpected newline in string (string-no-newline)
3:2     Unexpected newline in string (string-no-newline)
4:1     Unexpected newline in string (string-no-newline)
5:7     Unexpected newline in string (string-no-newline)
6:17    Unexpected newline in string (string-no-newline)
7:2     Unexpected newline in string (string-no-newline)
8:1     Unexpected newline in string (string-no-newline)
9:7     Unexpected newline in string (string-no-newline)
10:17   Unexpected newline in string (string-no-newline)
11:2    Unexpected newline in string (string-no-newline)
```
